### PR TITLE
added implementation for data access classes and interface for group, have yet to throw exceptions

### PR DESCRIPTION
**Pull request by sopleee**
group data will come in strings that need to be parsed at use-case level. the entities will each need to implement toString() and fromString() methods to parse

### CODE REVIEW

**Review by projectavi — APPROVED** [review]
Good job on the implementation of groupdata. I appreciate the number of comments you've added to the code to improve readability and ease of understanding - made it much easier to read over.

File: src/main/java/DataAccess/GroupDataAccess.java [hunk 1]
```diff
@@ -1,26 +1,65 @@
...
+        // this parses the json file that already exists into a hashmap of id and info strings
+//        this.jsonFile = new FileWriter(jasonPath);
+        this.jasonFile = new File(jasonPath);
+        FileReader reader = new FileReader(jasonFile);
+        JSONParser jsonParser = new JSONParser();
+        Object obj = jsonParser.parse(reader);
+        ((JSONArray) obj).forEach( grpObj -> parseGroupObject( (JSONObject) grpObj ) );
+    }
+    private void parseGroupObject(JSONObject groupObject){
+        //Get employee object within list
```
Comment (kunzhi-yu) [inline src/main/java/DataAccess/GroupDataAccess.java]: Remove this line

**Review by mishaalkandapath — DISMISSED** [review]
please work on a new branch for each feature related to the database. 
Cannot approve for that reason.
Move your changes to a new branch and reset this branch

**Review by kunzhi-yu — APPROVED** [review]
I look forward to working on testing if our dataaccess layer works. This was very helpful in comparing how group and user data structures work. You will need to work out what happens when the file does not yet exist.

**Pull request closed without merging**